lib/actions/ui.js

```diff
@@ -202,37 +204,69 @@ export function moveTo(i) {
   };
 }
 
+function getDefaultJsEditor() {
+  const regKey = '\\JSFile\\Shell\\Edit\\Command';
+  new Registry({
+    hive: 'HKCR',
+    key: regKey
+  }).get(Registry.DEFAULT_VALUE, (err, entry) => {
+    if (err) {
+      // In case of error, return default editor
+      return 'notepad.exe';
+    }
+    // Default is "C:\Windows\System32\Notepad.exe %1"
+    entry = entry.substring(0, entry.indexOf(' %1'));
+    entry = entry.substring(entry.lastIndexOf('\\'), entry.length);
+    return entry;
+  });
+}
+
 export function showPreferences() {
   const isWin = process.platform === 'win32';
   // eslint-disable-next-line no-template-curly-in-string
-  const command = isWin ? ' start notepad "%userprofile%\\.hyper.js"' : ' bash -c \'exec env ${EDITOR:=nano} ~/.hyper.js\'';
-  const message = [];
-  message.push(isWin ?
-    ' echo Attempting to open ^%userprofile^%\\.hyper.js with notepad' :
-    ' echo Attempting to open ~/.hyper.js with your \\$EDITOR');
-  message.push(' echo If it fails, open it manually with your favorite editor!');
+  let command = ' bash -c \'exec env ${EDITOR:=nano} ~/.hyper.js\'';
```

Comment:
In my opinion, `shell.showIteminFolder` should be used across all platforms. I'm sure there are a lot of unix users that don't have an editor set and will have to screw around with nano because they are unaware of this line.

End of the day, I don't even know if we should check the registry on windows; this is such an infrequent command it's unfortunate it requires so much complication. My recommendation is to overhaul all of this with a single `shell.showItemInFolder` for all platforms.